homeworks/src/main/java/hw31/ApplicationDemo.java

```diff
@@ -0,0 +1,11 @@
+package hw31;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication(scanBasePackages = "hw31")
+public class ApplicationDemo {
```

Comment:
It is not clear to me, why are you keep adding all the homeworks/changes to a single PR? You were supposed to add only the classes, that are required in the task. I remember we discussed it. It is hard for anyone who should review your code to find the required classes in this mess.